sklearn/feature_selection/univariate_selection.py

```diff
@@ -159,12 +164,13 @@ def _chisquare(f_obs, f_exp):
 
 
 def chi2(X, y):
-    """Compute chi-squared statistic for each class/feature combination.
+    """Compute chi-squared statistic between each class and non-negative feature
```

Comment:
[pep 0257](https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings) says only one line, so I'd leave it as before. The addition below is a good clarification, though.